website_sale_category_empty/models/product_public_category.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class ProductPublicCategory(models.Model):
+    _inherit = 'product.public.category'
+
+    product_ids = fields.Many2many(
+        comodel_name='product.template',
+        string='Products',
+        relation='product_public_category_product_template_rel',
```

Comment:
Sweet - can you add a note of the specific relation that you're hooking into please? This could be a pain point in migrations